qemu/tests/image_creation_luks_with_non_utf8_secret.py

```diff
@@ -0,0 +1,44 @@
+import os
+import re
+
+from avocado.utils import process
+
+from virttest import data_dir
+from virttest import utils_misc
+
+
+def run(test, params, env):
+    """
+    Negative test.
+    Luks image creation with non_utf8_secret:
+    1. It should be failed to create the image.
+    2. The error information should be corret.
+       e.g. Data from secret sec0 is not valid UTF-8
+
+    :param test: Qemu test object.
+    :param params: Dictionary with the test parameters.
+    :param env: Dictionary with test environment.
+    """
+    image_stg_name = params["image_name_stg"]
+    root_dir = data_dir.get_data_dir()
+    image_stg_path = utils_misc.get_path(root_dir, image_stg_name)
+    if os.path.exists(image_stg_path):
+        os.remove(image_stg_path)
+    err_info = params["err_info"]
+    non_utf8_secret_file = utils_misc.get_path(root_dir, "non_utf8_secret")
```

Comment:
```suggestion
    non_utf8_secret_file = os.path.join(data_dir.get_tmp_dir(), "non_utf8_secret")
```
We could put the non_utf8_secret file into the avocado temporary directory, then this temporary directory will be removed automatically after testing so that we could not call the line 42~44 again.

Reply:
updated, thanks.